app/views/coronavirus_form/contact_details.html.erb

```diff
@@ -0,0 +1,55 @@
+<% content_for :title do %><%= t('coronavirus_form.contact_details.title') %><% end %>
+<% content_for :meta_tags do %>
+  <meta name="description" content="<%= t('coronavirus_form.contact_details.title') %>" />
+<% end %>
+
+<% content_for :back_link do %>
+  <%= render "govuk_publishing_components/components/back_link", { href: previous_path } %>
+<% end %>
+
+<%= render "govuk_publishing_components/components/title", {
+  title:t("coronavirus_form.contact_details.title"),
+  margin_top: 0
+} %>
+
+<%= form_tag({},
+  "data-module": "track-coronavirus-form-business-contact-details",
+  "data-question-key": "contact_details",
+  "id": "contact_details"
+) do %>
+<%= render "govuk_publishing_components/components/input", {
```

Comment:
do we indent this, given the do?

Reply:
We haven't elsewhere, so I left it like that for consistency.